keps/sig-node/asier/README.md

```diff
@@ -0,0 +1,530 @@
+
+highlighting any additional information provided beyond the standard KEP
+
+template.
+
+  
+
+Ensure the TOC is wrapped with
+
+<code>&lt;!-- toc --&rt;&lt;!-- /toc --&rt;</code>
+
+tags, and then generate with `hack/update-toc.sh`.
+
+-->
+
+  
+
+<!-- toc -->
+
+- [KEP-XXXXX: IMA namespace support inside containers](#kep-xxxxx-ima-namespace-support-inside-containers)
+  - [Release Signoff Checklist](#release-signoff-checklist)
+  - [Summary](#summary)
+  - [Motivation](#motivation)
+    - [Goals](#goals)
+    - [Non-Goals](#non-goals)
+  - [Proposal](#proposal)
+    - [User Stories (Optional)](#user-stories-optional)
+      - [Story 1](#story-1)
+      - [Story 2](#story-2)
+      - [Story 3](#story-3)
+    - [Notes/Constraints/Caveats (Optional)](#notesconstraintscaveats-optional)
+    - [Risks and Mitigations](#risks-and-mitigations)
+  - [Design Details](#design-details)
+    - [Linux kernel](#linux-kernel)
+    - [Runtime specification](#runtime-specification)
+    - [CRI API](#cri-api)
+    - [Kubernetes pod resource](#kubernetes-pod-resource)
+    - [Monitoring and alerting](#monitoring-and-alerting)
+    - [Test Plan](#test-plan)
+    - [Graduation Criteria](#graduation-criteria)
+      - [GA](#ga)
+    - [Upgrade / Downgrade Strategy](#upgrade--downgrade-strategy)
+    - [Version Skew Strategy](#version-skew-strategy)
+  - [Production Readiness Review Questionnaire](#production-readiness-review-questionnaire)
+  - [Implementation History](#implementation-history)
+  - [Drawbacks](#drawbacks)
+  - [Alternatives](#alternatives)
+  - [Infrastructure Needed (Optional)](#infrastructure-needed-optional)
+
+<!-- /toc -->
+
+  
+
+## Release Signoff Checklist
+
+  
+
+<!--
+
+**ACTION REQUIRED:** In order to merge code into a release, there must be an
+
+issue in [kubernetes/enhancements] referencing this KEP and targeting a release
+
+milestone **before the [Enhancement Freeze](https://git.k8s.io/sig-release/releases)
+
+of the targeted release**.
+
+  
+
+For enhancements that make changes to code or processes/procedures in core
+
+Kubernetes—i.e., [kubernetes/kubernetes], we require the following Release
+
+Signoff checklist to be completed.
+
+  
+
+Check these off as they are completed for the Release Team to track. These
+
+checklist items _must_ be updated for the enhancement to be released.
+
+-->
+
+  
+
+Items marked with (R) are required *prior to targeting to a milestone / release*.
+
+<!--
+
+**Note:** This checklist is iterative and should be reviewed and updated every time this enhancement is being considered for a milestone.
+
+-->
+
+  
+
+[kubernetes.io]:  https://kubernetes.io/
+
+[kubernetes/enhancements]:  https://git.k8s.io/enhancements
+
+[kubernetes/kubernetes]:  https://git.k8s.io/kubernetes
+
+[kubernetes/website]:  https://git.k8s.io/website
+
+  
+
+## Summary
+
+  
+
+IMA namespaces allow to check the file integrity. This proposal adds file integrity inside containers deployed in kubernetes.
```

Comment:
"the file" --> which file? The rootfs?

Reply:
This is first of all about regular files I suppose @asierHuawei

Reply:
Yes, regular files inside the container. I will change the wording to be more explicit.